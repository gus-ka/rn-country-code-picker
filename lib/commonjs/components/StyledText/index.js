"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var _react = _interopRequireDefault(require("react"));
var _reactNative = require("react-native");
var _styles = _interopRequireDefault(require("./styles"));
function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
const StyledText = _ref => {
  let {
    children,
    style
  } = _ref;
  return /*#__PURE__*/_react.default.createElement(_reactNative.Text, {
    style: [style, _styles.default.text]
  }, children, " ");
};
var _default = StyledText;
exports.default = _default;
//# sourceMappingURL=index.js.map