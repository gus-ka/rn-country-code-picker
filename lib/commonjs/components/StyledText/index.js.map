{"version":3,"names":["_react","_interopRequireDefault","require","_reactNative","_styles","obj","__esModule","default","StyledText","_ref","children","style","createElement","Text","styles","text","_default","exports"],"sources":["index.tsx"],"sourcesContent":["import React from 'react';\nimport { Text } from 'react-native';\nimport { IStyledTextProps } from '../../types';\nimport styles from './styles';\n\nconst StyledText = ({ children, style }: IStyledTextProps) => {\n  return <Text style={[style, styles.text]}>{children} </Text>;\n};\n\nexport default StyledText;\n"],"mappings":";;;;;;AAAA,IAAAA,MAAA,GAAAC,sBAAA,CAAAC,OAAA;AACA,IAAAC,YAAA,GAAAD,OAAA;AAEA,IAAAE,OAAA,GAAAH,sBAAA,CAAAC,OAAA;AAA8B,SAAAD,uBAAAI,GAAA,WAAAA,GAAA,IAAAA,GAAA,CAAAC,UAAA,GAAAD,GAAA,KAAAE,OAAA,EAAAF,GAAA;AAE9B,MAAMG,UAAU,GAAGC,IAAA,IAA2C;EAAA,IAA1C;IAAEC,QAAQ;IAAEC;EAAwB,CAAC,GAAAF,IAAA;EACvD,oBAAOT,MAAA,CAAAO,OAAA,CAAAK,aAAA,CAACT,YAAA,CAAAU,IAAI;IAACF,KAAK,EAAE,CAACA,KAAK,EAAEG,eAAM,CAACC,IAAI;EAAE,GAAEL,QAAQ,EAAC,GAAO,CAAC;AAC9D,CAAC;AAAC,IAAAM,QAAA,GAEaR,UAAU;AAAAS,OAAA,CAAAV,OAAA,GAAAS,QAAA"}