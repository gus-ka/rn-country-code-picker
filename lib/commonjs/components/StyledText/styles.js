"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;
var _reactNative = require("react-native");
var _default = _reactNative.StyleSheet.create({
  text: {
    fontSize: 16,
    lineHeight: 24
  }
});
exports.default = _default;
//# sourceMappingURL=styles.js.map