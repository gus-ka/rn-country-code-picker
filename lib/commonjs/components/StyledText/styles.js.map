{"version":3,"names":["_reactNative","require","_default","StyleSheet","create","text","fontSize","lineHeight","exports","default"],"sources":["styles.ts"],"sourcesContent":["import { StyleSheet } from 'react-native';\n\nexport default StyleSheet.create({\n  text: {\n    fontSize: 16,\n    lineHeight: 24,\n  },\n});\n"],"mappings":";;;;;;AAAA,IAAAA,YAAA,GAAAC,OAAA;AAA0C,IAAAC,QAAA,GAE3BC,uBAAU,CAACC,MAAM,CAAC;EAC/BC,IAAI,EAAE;IACJC,QAAQ,EAAE,EAAE;IACZC,UAAU,EAAE;EACd;AACF,CAAC,CAAC;AAAAC,OAAA,CAAAC,OAAA,GAAAP,QAAA"}