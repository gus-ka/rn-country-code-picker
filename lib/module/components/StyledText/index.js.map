{"version":3,"names":["React","Text","styles","StyledText","_ref","children","style","createElement","text"],"sources":["index.tsx"],"sourcesContent":["import React from 'react';\nimport { Text } from 'react-native';\nimport { IStyledTextProps } from '../../types';\nimport styles from './styles';\n\nconst StyledText = ({ children, style }: IStyledTextProps) => {\n  return <Text style={[style, styles.text]}>{children} </Text>;\n};\n\nexport default StyledText;\n"],"mappings":"AAAA,OAAOA,KAAK,MAAM,OAAO;AACzB,SAASC,IAAI,QAAQ,cAAc;AAEnC,OAAOC,MAAM,MAAM,UAAU;AAE7B,MAAMC,UAAU,GAAGC,IAAA,IAA2C;EAAA,IAA1C;IAAEC,QAAQ;IAAEC;EAAwB,CAAC,GAAAF,IAAA;EACvD,oBAAOJ,KAAA,CAAAO,aAAA,CAACN,IAAI;IAACK,KAAK,EAAE,CAACA,KAAK,EAAEJ,MAAM,CAACM,IAAI;EAAE,GAAEH,QAAQ,EAAC,GAAO,CAAC;AAC9D,CAAC;AAED,eAAeF,UAAU"}