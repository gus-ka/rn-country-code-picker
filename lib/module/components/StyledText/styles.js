import { StyleSheet } from 'react-native';
export default StyleSheet.create({
  text: {
    fontSize: 16,
    lineHeight: 24
  }
});
//# sourceMappingURL=styles.js.map