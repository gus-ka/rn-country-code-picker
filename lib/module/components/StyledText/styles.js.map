{"version":3,"names":["StyleSheet","create","text","fontSize","lineHeight"],"sources":["styles.ts"],"sourcesContent":["import { StyleSheet } from 'react-native';\n\nexport default StyleSheet.create({\n  text: {\n    fontSize: 16,\n    lineHeight: 24,\n  },\n});\n"],"mappings":"AAAA,SAASA,UAAU,QAAQ,cAAc;AAEzC,eAAeA,UAAU,CAACC,MAAM,CAAC;EAC/BC,IAAI,EAAE;IACJC,QAAQ,EAAE,EAAE;IACZC,UAAU,EAAE;EACd;AACF,CAAC,CAAC"}