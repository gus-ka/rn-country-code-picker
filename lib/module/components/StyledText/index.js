import React from 'react';
import { Text } from 'react-native';
import styles from './styles';
const StyledText = _ref => {
  let {
    children,
    style
  } = _ref;
  return /*#__PURE__*/React.createElement(Text, {
    style: [style, styles.text]
  }, children, " ");
};
export default StyledText;
//# sourceMappingURL=index.js.map