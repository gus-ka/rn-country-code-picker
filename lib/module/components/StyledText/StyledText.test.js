import React from 'react';
import { render } from '@testing-library/react-native';
import StyledText from './';
test('renders correctly', () => {
  const text = 'Test Text';
  const {
    toJSON,
    getByText
  } = render( /*#__PURE__*/React.createElement(StyledText, null, text));
  const rendered = toJSON();
  expect(getByText(`${text} `)).toBeTruthy();
  expect(rendered).toMatchSnapshot();
});
//# sourceMappingURL=StyledText.test.js.map