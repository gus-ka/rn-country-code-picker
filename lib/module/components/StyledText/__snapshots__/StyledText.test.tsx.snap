// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Array [
      undefined,
      Object {
        "fontSize": 16,
        "lineHeight": 24,
      },
    ]
  }
>
  Test Text
   
</Text>
`;
